Source/URLDispatcher.spoon/init.lua

```diff
@@ -0,0 +1,83 @@
+--- === URLDispatcher ===
+---
+--- Flexible URL handling
```

Comment:
This line is what's shown on the API docs index for Spoons ([http://www.hammerspoon.org/Spoons/](http://www.hammerspoon.org/Spoons/)) so I think it should be a little more descriptive. Maybe:

`Route URLs to different applications with pattern matching`?